packages/widget-utils/src/bootstrap/bootstrapWidgets.js

```diff
@@ -28,16 +25,21 @@ const initializeWidget = async (backendUrl, container) => {
   const key = getWidgetKey(container)
   const widgetConfig = await executeRequest(`${backendUrl}/nice2/rest/widget/configs/${key}`)
     .then(enhanceExtractedBody)
-    .then(response => handleRequestError(response, key))
+    .then(response => handleRequestError(backendUrl, response, key))
     .then(response => response?.body)
     .catch(error => {
-      // TODO: TOCDEV-4757 - try log remote
-      consoleLogger.logError(`Could not fetch widget config '${key}'.`, error)
+      remoteLogger.logException(backendUrl, `Could not fetch widget config '${key}'.`, error)
     })
 
   if (widgetConfig) {
     const {appName, packageName, locale, config} = widgetConfig
-    await loadScriptAsync(`${backendUrl}/js/tocco-${packageName}/dist/index.js`)
+
+    try {
+      await loadScriptAsync(`${backendUrl}/js/tocco-${packageName}/dist/index.js`)
+    } catch (error) {
+      remoteLogger.logException(backendUrl, `Could not fetch package 'tocco-${packageName}'.`, error)
+      return
+    }
```

Comment:
should we display something to the user if a widget couldn't be loaded  (e.g. an exclamation mark in the widget container) or should it just remain empty/invisible?

Reply:
Hmm good question 🤔

Reply:
Same question for errors during the render... or in general.
I will merge this as is and start the discussion with Juan next week.